source/Classes/004.ManagedIdentityToken.ps1

```diff
@@ -0,0 +1,120 @@
+
+
+class ManagedIdentityToken {
+
+    [SecureString]$access_token
+    [DateTime]$expires_on
+    [int]$expires_in
+    [string]$resource
+    [string]$token_type
+    hidden [bool]$linux = $IsLinux
```

Comment:
Perhaps this shouldn't be a check for 1 OS type unless this is definitely an only linux and not also MacOS issue to check for & not use the `$Is` automatic varibales for checking and use the v5 compatable `[Environment]::OSVersion.Platform` instead